src/Tizen.NUI.Components/Controls/Progress.cs

```diff
@@ -223,6 +227,38 @@ public string BufferImageURL
             set => bufferImage.ResourceUrl = value;
         }
 
+        /// <summary>
+        /// The property to get/set the indeterminate image.
+        /// </summary>
+        /// <exception cref="NullReferenceException">Thrown when setting null value.</exception>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string IndeterminateImage
```

Comment:
According to other Progress's API,
the string path is expressed as `XXXURL`. (TrackImageURL, ProgressImageURL).
For the naming consistency, this API should be `IndeterminateImageURL`.

But the `URL` is against the c# coding convention. (`Url` is the correct.)

Could you please make all APIs to have correct name?
* System.Uri TrackImageUri
* System.Uri ProgressImageUri
* System.Uri IndeterminateImageUri

Reply:
As we discuss it today, I'll change the name to `IndeterminateImageUrl`.
Do you think we should add `TrackImageUrl` or `ProgressImageUrl` and deprecate the current one?

Reply:
I think it'll be better to deprecate existing `XXXURL` and add new `XXXUrl`. Thank you!

Reply:
Okay, great. I'll add new ones together when I proceed ACR for other Slider public properties. :)